README.md

```diff
@@ -61,7 +70,7 @@ Execute the commands below to build from source.
 
 As an open source project, Ballerina welcomes contributions from the community. 
 
-You can also check for [open issues](https://github.com/ballerina-platform/module-ballerina-log/issues) that interest you. We look forward to receiving your contributions.
+You can also check for [open issues](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Flog) that interest you. We look forward to receiving your contributions.
```

Comment:
According to https://github.com/ballerina-platform/module-ballerina-http/pull/99/files this line has to be removed.

Reply:
Removed in 82d276f